app/enterprise/0.33-x/rbac/overview.md

```diff
@@ -124,7 +124,7 @@ entities permissions; a role may be associated with more than one endpoint
 or entity permission. Roles are defined by a human-readable name, such as
 "read-only".
 
-#### Role - Endpoints
+### Role - Endpoints
```

Comment:
In theory, we'd want #### here, because "Role - Endpoints" is a sub-topic under "Role" - but in practice, I think we'd only get that level of considerate heading level setting is we are doing it "by hand" - am I right?

Reply:
I have generally followed the structure as it was before, where these headers were all siblings.
So yeah, to get them as sub-topic, it needs to be done more intelligently/by hand.

Reply:
OK, we'll save the "by hand" tuneups for later :)